### PR TITLE
Fix: Submodule diffs are misaligned

### LINKED ISSUE #4328: Submodule diffs are misaligned
# What?

1. __Primary__: Prevent center-aligned filenames in submodule diff.
2. _Secondary_: If submodule diffs were previously ignored during styling, then reconsider the styling.

# Why?

A center-aligned file path listing is not as easy to scan as is a left-aligned file path listing, because similarities in paths are not aligned as they are in a left-aligned file path listing.

# Where?

https://github.com/TACC/Core-CMS/pull/216/commits/c8b7cbe270ad083dd11d3f81da35cb281fe3d56f

<details><summary>Screenshot while using Github Refined</summary>

Github Refined

</details>

<details><summary>Screenshot while using Github</summary>

Github

</details>

# How?

__Offending Style__:
https://github.com/sindresorhus/refined-github/blob/8c1ab3125fcdb5edf4d78e86bca2da30da97ca45/source/refined-github.css#L99-L103

__Suggestions__: None

### PR DESCRIPTION
**Pull request by harshgoel05**
Fixes #4328 

## Test URLs
[Link to test this fix : TACC/Core-CMS@c8b7cbe](https://github.com/TACC/Core-CMS/commit/c8b7cbe270ad083dd11d3f81da35cb281fe3d56f)

[Link to test that it doesn't break 1639](https://github.com/webpack/webpack/pull/6446/files#diff-4cb0a7248ed91703f66f261958573976e311153c7273acaa2f6960be6dd84e98)

## Screenshot
Before:
image

After:
image

How:
Added overwrite to shift the table by `text-align: center;` to `text-align: left;`

### CODE REVIEW

**Comment by yakov116** [comment]
Did you check that it did not break https://github.com/sindresorhus/refined-github/pull/1639?

**Comment by harshgoel05** [comment]
I've updated my PR with the link. 
Also, I'm sorry I will take care of that next time. (I had disabled it for a while so didn't get a warning)

File: source/refined-github.css [hunk 1]
```diff
@@ -101,6 +101,9 @@
 	padding: 54px;
 	text-align: center;
 }
+:root .file .data table {
+	text-align: left !important;
```
Comment (fregante) [inline source/refined-github.css:105]: I don’t think we need important. Also the lint is failing and needs to be fixed

**Review by fregante — COMMENTED** [review]
Thank you for the PR!

**Comment by harshgoel05** [comment]
@fregante Fixed those! Thanks for feedback :)

**Comment by harshgoel05** [comment]
@fregante Fixed those! Thanks for the feedback :)

**Comment by fregante** [comment]
Thank you for the fix, @harshgoel05!

**Pull request merged**